composer.json

```diff
@@ -19,11 +19,13 @@
         "php": "7.4.*",
         "ext-json": "*",
         "doctrine/annotations": "^1.10.0",
+        "symfony/config": "4.4.*",
         "symfony/dependency-injection": "4.4.*",
         "symfony/event-dispatcher": "4.4.*",
         "symfony/http-foundation": "4.4.*",
         "symfony/http-kernel": "4.4.*",
-        "symfony/config": "4.4.*"
+        "symfony/options-resolver": "^5.0",
+        "webonyx/graphql-php": "^0.13.0"
```

Comment:
As @Fabex said this should not be `require`d for all users.
But I'd rather say that a simple `suggest` for `webonyx/graphql-php` would be enough, instead of a "plugin system"

Reply:
Done